docs/web/docs/1-getting_started/1-getting_started.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Install and Configure
+title: Getting started
```

Comment:
I had an issue with the order of the Table of contents (see below) but it seems to work when this file is named `00-getting_started.mdx` instead of `1-getting_started.mdx`

Reply:
Thanks, I'll take care of it. Do you mind pointing me to instructions on how to run it locally, if you have something handy, otherwise I'll try to figure it out.